src/kit/ResponsiveGroup.tsx

```diff
@@ -55,12 +55,18 @@ const ResponsiveGroup: React.FC<Props> = ({
       },
       { accumulatedWidth: 0, visible: 0 },
     );
+
     setNumVisible((prev) => {
-      if (prev !== visible) onChange?.(visible);
-      return visible;
+      if (prev !== visible) return visible;
+
+      return prev;
```

Comment:
i don't think you need a functional update for this value anymore -- these are numbers, so `Object.is` will return true if you pass the same value in, no?

Reply:
Yeah, I think you should be able to just do `setNumVisible(visible)`